javascript/sdk/lib/controllers/partner.ts

```diff
@@ -115,7 +116,7 @@ export default class PartnerController {
   async webhookGenerate(
     partnerCode: ControlCode,
     options: RequestOptions = {}
-  ): Promise<StatusResponse> {
+  ): Promise<GeneratedWebhook> {
     const response = await this.#client.requestWithAuth(
       `/partner/suppress/${partnerCode}`,
```

Comment:
also we moved to enum names and I think ControlCode is still the int value?

Reply:
fixed!